scripts/runtests.sh

```diff
@@ -3,7 +3,7 @@
 provider2openssl() {
     echo
     echo "Testing oqsprovider->oqs-openssl interop for $1:"
-    $OQS_PROVIDER_TESTSCRIPTS/scripts/oqsprovider-certgen.sh $1 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqsprovider-cmssign.sh $1 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqs-openssl-certverify.sh $1 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqs-openssl-cmsverify.sh $1
+    $OQS_PROVIDER_TESTSCRIPTS/scripts/oqsprovider-certgen.sh $1 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqsprovider-cmssign.sh $1 md5 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqs-openssl-certverify.sh $1 && $OQS_PROVIDER_TESTSCRIPTS/scripts/oqs-openssl-cmsverify.sh $1
```

Comment:
Because it's the most retro-alg I could find :-) Seriously: `sha512` and "nothing" (collecting all data to be passed as-is) are anyway used/tested as part of regular operations, e.g., certgen, so I was looking for something _really_ different. Other proposals welcome. Just no full matrix, please: Testing anyway already is getting ecologically unfriendly (read: Takes pretty long).

Reply:
Ok, I just wondered if there are some interop/legacy requirements to test with md5, as it shouldn't be really used in practice. Maybe use sha3 instead which will be more common?

Reply:
Boring. But so done.